mixer/README.md

```diff
@@ -1,5 +1,24 @@
 # Mixer
 
+## Mixer's future
+
+Because of the extensive feedback we received from the community about the
+difficulties encountered with Mixer, we are making a substantial change to
+Istio's architecture. By taking advantage of the work being done in the
+[WebAssembly](https://webassembly.org) community, we are working to move the
+functionality that formerly lived in Mixer into the Envoy proxy itself.
+
+It reduces the complexity of installation (fewer components, fewer CRDs),
+makes it easier to run Istio, and importantly it allows integrations in a more
+standard manner.
+
+To read more about the changes (which we're tentatively calling Extensions v2),
+please see
+[this doc](
+https://docs.google.com/document/d/1x5XeKWRdpFPAy7JYxiTz5u-Ux2eoBQ80lXT6XYjvUuQ/edit#heading=h.8kpssnjs5pqw)
```

Comment:
is this document open to public or requires you to be part of a google group?

Reply:
Hey, @nrjpoddar, nice to be here! This requires you to be part of the group that gives you access to all design docs. Should I put a note here?

Reply:
It would be good to add a note, thanks!